app/Models/Log.php

```diff
@@ -52,6 +52,51 @@ public function setLogDataAttribute($value)
         $this->attributes['log_data'] = serialize($value);
     }
 
+    public function dataForDisplay()
+    {
+        if ($this->log_operation === 'LOG_TOPIC_TYPE') {
+            switch ($this->log_data['type']) {
```

Comment:
can use `$translationString = match ($this->log_data['type']) { ... }`

...and why is it called `translationString` while the rest of function calls it `translationKey`